src/components/common/BcRegComments.vue

```diff
@@ -0,0 +1,311 @@
+<template>
+  <div>
+    <template v-if="isBenefit">
```

Comment:
I think this is fine not being in the resource file, it's fairly dynamic and requires quite a bit of html